scripts/run_on_kube.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/python
```

Comment:
I don't know of a way offhand to prevent the job controller from re-starting pods when they fail.